.github/workflows/generate-files.yml

```
@@ -31,9 +31,7 @@ jobs:
 
       - name: Check for changes
         run: |
-          if git diff --exit-code --ignore-space-change --ignore-all-space --ignore-cr-at-eol; then
-            echo "Generated Go files are up-to-date."
-          else
-            echo "::error::Generated files are not up-to-date. Please run 'make generate' locally and commit any changes."
+          if [[ -n $(git status --porcelain) ]]; then
+            echo "::error::Generated files are not up-to-date (including deleted files). Please run 'make generate' locally and commit any changes."
             exit 1
-          fi
+          fi
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)